comment-updater-headless/src/main/kotlin/HeadlessConfig.kt

```diff
@@ -0,0 +1,4 @@
+object HeadlessConfig {
+    val OUTPUT_DIR_PATH = "/Users/Ivan.Pavlov/IdeaProjects/CommentUpdater1/dataset"
```

Comment:
why it is hardcoded here? it would be better if a user could pass these paths as arguments

Reply:
This file should be deleted (now it is). This variables aren't used anywhere.

Reply:
The next possible useful plugin - deleting outdated files (with no used variables, for examples) =)